modules/cluster-logging-deploy-multitenant.adoc

```diff
@@ -5,26 +5,29 @@
 [id="cluster-logging-deploy-multitenant_{context}"]
 = Installing OpenShift Logging into a multitenant network
 
-If you are deploying OpenShift Logging into a cluster that uses multitenant isolation mode, projects are isolated from other projects. As a result, network traffic is not allowed between pods or services in different projects.
+If you deploy OpenShift Logging into a cluster that uses multitenant isolation mode, projects are isolated from other projects. As a result, network traffic is not allowed between pods or services in different projects.
 
 Because the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator are installed in different projects, you must explicitly allow access between the  `openshift-operators-redhat` and `openshift-logging` projects. How you allow this access depends on how you configured multitenant isolation mode.
 
 .Procedure
 
-To allow traffic between the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator, perform one of the following:
+To allow traffic between the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator, perform either of the following actions:
 
-* If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *Multitenant* mode, use the following command to join the two projects:
-+
-For example:
+* If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *Multitenant* mode, join the two projects. For example:
 +
 [source,terminal]
 ----
 $ oc adm pod-network join-projects --to=openshift-operators-redhat openshift-logging
 ----
 
-* If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *NetworkPolicy* mode, create a network policy object in the `openshift-logging` namespace that allows ingress from the `openshift-operators-redhat` project to the `openshift-logging` project.
+* If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *NetworkPolicy* mode, set a label on the `openshift-operators-redhat` namespace. For example:
```

Comment:
fwiw, At some point, this won't be strictly necessary if enabled and available in OCP: https://kubernetes.io/docs/concepts/services-networking/network-policies/#targeting-a-namespace-by-its-name

Reply:
> fwiw, At some point, this won't be strictly necessary
I wonder, what's the best way to track whether and when we downstream this change? Any suggestions?